src/controllers/room.controller.js

```diff
@@ -0,0 +1,82 @@
+import { Room } from '../models/Room.models.js';
+
+export const getRooms = async (req, res) => {
+  res.send(await Room.findAll());
+};
+
+export const getOneRoom = async (req, res) => {
+  const { id } = req.params;
+  const room = await Room.findByPk(id);
+
+  if (!room) {
+    res.status(404).send({ message: 'cant get a room' });
+
+    return;
+  }
+  res.status(200);
```

Comment:
In your `getOneRoom` function, you are setting the status code to 200 but you are not ending the response. This could lead to hanging requests as the response is never closed. You should send the response in the same line you set the status code. It would look like this: `res.status(200).send(room);`